interface/patient_file/summary/add_edit_issue.php

```diff
@@ -806,8 +808,18 @@ function divclick(cb, divid) {
                     </div>
                 </div>
                 <!-- End of reaction -->
+                <!-- Verification Status for Medication Allergy -->
+                <div class="form-group" id='row_verification'>
```

Comment:
Also, this should only show in the allergy UI (like the Reaction), however, it is showing in all other lists items UI (medication, medical problems, surgeries, etc.)